index.html

```diff
@@ -170,7 +184,7 @@ <h1 class="contact">Contact me</h1>
      project that needs coding.<br> 
     I'd love to help with it!</p>
     <div class="forms">
-    <form method="post" action="">
+    <form method="post" action="https://formspree.io/f/xvojodzg">
         <input type="text" name="name" id="name"  placeholder="Lorem Ipsum" maxlength="30">
         <input type="email" name="email" id="email" placeholder="loremipsum@gmail.com" required>
         <textarea  name="message" id="message" placeholder="Write your message here..." cols="30" rows="10" maxlength="500" required ></textarea>
```

Comment:
Please change your placeholder font-color to the color given in the figma design :

Figma design :
![image](https://github.com/Surafels/Portfolio_Mobile/assets/20537712/dbc5d6ee-8ad8-4986-a7b3-4e587e241ac8)

Your design :
![image](https://github.com/Surafels/Portfolio_Mobile/assets/20537712/45002d3d-af3a-4489-9057-ae9448a9838f)